internal/controller/proxmoxcluster_controller_test.go

```diff
@@ -201,6 +205,101 @@ var _ = Describe("Controller Test", func() {
 	})
 })
 
+var _ = Describe("Proxmox ClusterReconciler", func() {
```

Comment:
if you want a different context, 
Please rename accordingly.

e.g. `External Credentials Tests`

Reply:
Renamed as suggested.
Thanks.